test-app/app/models/big-hat.js

```diff
@@ -2,5 +2,5 @@ import Hat from './hat';
 import { hasMany } from '@ember-data/model';
 
 export default class extends Hat {
-  @hasMany('soft-material', { async: false }) materials;
+  @hasMany('soft-material', { async: false, inverse: 'hat' }) materials;
```

Comment:
ED 5 requires all async/inverse/as relationship options to be explicitly defined - no more defaults and assumptions